contracts/core/lib/auction-price-libraries/LinearAuctionPriceCurve.sol

```diff
@@ -78,25 +78,50 @@ contract LinearAuctionPriceCurve {
      *
```

Comment:
I would add an in-depth comment that describes this. I might even try to do a graphic/chart in code format